test/In.ProjectEKA.HipServiceTest/DataFlow/DataFlowTest.cs

```diff
@@ -10,34 +11,40 @@ namespace In.ProjectEKA.HipServiceTest.DataFlow
     using Xunit;
     using HipService.Consent;
     using HipService.MessagingQueue;
-    
+    using Microsoft.Extensions.Options;
     using DataFlowService = HipService.DataFlow.DataFlow;
-    
+
     [Collection("Data Flow Tests")]
     public class DataFlowTest
     {
         private readonly Mock<IDataFlowRepository> dataFlowRepository = new Mock<IDataFlowRepository>();
         private readonly Mock<IConsentRepository> consentRepository = new Mock<IConsentRepository>();
+        private readonly Mock<ILinkDataRepository> linkDataRepository = new Mock<ILinkDataRepository>();
         private readonly Mock<IMessagingQueueManager> messagingQueueManager = new Mock<IMessagingQueueManager>();
+        private readonly IOptions<DataFlowConfiguration> dataFlowConfiguration;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field 'dataFlowConfiguration' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)